packages/frontend-2/components/header/NavBar.vue

```diff
@@ -12,7 +12,7 @@
           />
           <PortalTarget name="navigation"></PortalTarget>
         </div>
-        <div class="flex items-center gap-1.5 sm:gap-2">
+        <div class="flex items-center gap-2.5 sm:gap-2">
```

Comment:
Let's increase to gap-4 instead of gap-2.5 to make it easier to tap with a fat finger

Reply:
Just bear in mind that there can be other items in the headerbar. I personally think we are losing too much space to in the headerbar here, which is why I kept it a lower figure, but happy to let you make the decision. 
<img width="296" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/1270049e-92dd-4821-b47b-df88a094df2d">

Reply:
good point, I only checked logged out where we don't have a notification bell. Let's just go with what you have here then